### ISSUE TITLE
Feature to deselect individual items in preview results

### ISSUE THREAD

**Issue by bendover22** [bleachbit/bleachbit#217]
I don't see a way to remove / protect individual items from preview results from being cleaned, other than uncheck a whole category and re-run entire preview.
Unchecking an entire category isn't always desirable.

Is it possible to add ability to right click a preview results item & "remove from cleaning list" or "don't clean this item," etc?  Lots of apps have this sort of feature.  That might be easier to implement than adding check boxes beside each preview item to select / deselect items.

I imagine it's very common to want to protect 1 or 2 items in category results.

If items are deselected / protected, they could be grayed out and / or italicized or change background color to show they're protected from cleaning.  

Some apps use check marks, 'X's' to indicate selected and deselected list items, but that may be more work than graying out, italicizing, etc.

Note:  green & red colors, alone, are difficult for color blind persons to see.

**Comment by ROCKNROLLKID** [comment]
Good idea. For now, you can just exclude the file in BleachBit settings, under whitelist.

**Comment by bendover22** [comment]
I checked on that.  Unfortunately, you can't copy / paste preview results you want exempted to the whitelist (items you weren't expecting it would find).  For several items, that turns into work.

If you know items ahead of time to always exclude, entering them once in the whitelist isn't too bad.  Unless there's a lot.

In Linux Mint 18.1 / Cinnamon, the best that BB would allow for "faster" whitelisting items off of Preview results, was highlighting only the path part of a Preview result, then "Add Item" in the whitelist.  That opens the system's default file manager, where (in Nemo & some others) you can paste a path just copied into the file mgr's location bar.

Then BB allows adding that path (if exists) from the file mgr  to the whitelist.  This isn't a lot faster than typing it, except for long paths.

Neither do I see a way to select multiple items from BB's Preview results & enter in the whitelist.  The BB "add file or folder to whitelist" GUI, nor the file mgr allow entering multiple paths, say comma separated or one item per line, etc.

**Comment by ROCKNROLLKID** [comment]
Yes, we have been wanting to change the GUI for awhile. I am not a programmer and the only person working on BB full time is Andrew, so you will need to be patient for such feature. It is a good idea, nonetheless.

**Comment by tstenner** [comment]
Maybe we should convert the resultbox from a textbox to something else with two levels for individual cleaners and a sublist of files each cleaner cleans. That way we could let someone select individual files and offer to add them to the whitelist in the context menu
